.vscode/launch.json

```diff
@@ -8,16 +8,25 @@
             "name": "api",
             "type": "debugpy",
             "request": "launch",
-            "program": "api/app/main.py",
+            "program": "${workspaceFolder}/backend/packages/wps-api/src/app/main.py",
             "console": "integratedTerminal",
-            "justMyCode": false
+            "justMyCode": false,
+            "cwd": "${workspaceFolder}/backend",
+            "python": "${workspaceFolder}/backend/.venv/bin/python",
+            "env": {
+                "PYTHONPATH": "${workspaceFolder}/backend/packages/wps-api/src:${workspaceFolder}/backend/packages/wps-shared/src",
```

Comment:
Is this `PYTHONPATH` env necessary? Seems odd that it points to the `wps-shared` package.

Reply:
It's not! Thanks, removed in: https://github.com/bcgov/wps/pull/4912/commits/ce9b9a2267f39a32718ad2332523616a16e4422b